components/Footer/Footer.module.scss

```diff
@@ -4,19 +4,17 @@
   }
 }
 .footerNav {
-  width: calc(100% - 30px);
+  width: 100%;
   background: var(--green);
   text-align: center;
-  position: fixed;
-  bottom: 1rem;
-  left: 15px;
   border-radius: var(--border-radius);
-  z-index: 9999;
+  padding: 2rem;
```

Comment:
Too many padding. Follow the copyright pattern. It's supposed to be 1rem.

Reply:
Done